livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/LiveStreamComment.kt

```diff
@@ -0,0 +1,78 @@
+package io.getstream.livestream.compose.ui
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import io.getstream.chat.android.client.models.name
+import io.getstream.chat.android.compose.state.messages.items.MessageItem
+import io.getstream.chat.android.compose.ui.common.MessageBubble
+import io.getstream.chat.android.compose.ui.common.avatar.UserAvatar
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.livestream.compose.getTimeAgo
+import java.text.SimpleDateFormat
+import java.util.Date
+
+/**
+ * View component to add custom live stream channels screen.
+ *
+ * @param modifier - Modifier for styling.
+ * @param messageItem - Data model for messageItem for each chat message row
+ */
+@Composable
+fun LiveStreamComment(
+    modifier: Modifier = Modifier,
+    messageItem: MessageItem
+) {
+    val context = LocalContext.current
+    Row(
+        modifier = modifier
+            .padding(start = 8.dp, bottom = 2.dp)
+            .width(300.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        UserAvatar(
+            modifier = Modifier
+                .padding(start = 8.dp, end = 8.dp)
+                .size(32.dp),
+            user = messageItem.message.user
+        )
+        Column {
+            Row {
```

Comment:
Same here, there is no vertical alignment for the title and the timestamp - this is why it's not centered and the timestamp is falling outside of the title space.

Reply:
https://github.com/GetStream/Android-Samples/pull/5#discussion_r689090710

Reply:
It should be described in our [documentation](https://getstream.io/chat/docs/sdk/android/compose/overview/).

We allow them to customize the theme, shapes, colors, typography (and will allow spacings in the future). We also allow to customize the behavior of items (overriding actions) and overriding slot APIs (e.g. actions and input views etc).

We also allow for full on override of certain components (e.g. LoadingView and EmptyView for the MessageList).

Here is not a matter of customizing this `Row` to allow the user to customize it much, as the `LiveStreamComment` is a default UI component and if our users want more custom behavior, they should override it.

It's a matter of the `Row` not having any modifiers or paramters, which makes the footnote/timestamp not center-aligned with the user name text element.